grpc.go

```diff
@@ -788,7 +790,12 @@ func (r *grpcRequest) setTraceHeader(s *step) error {
 		return err
 	}
 	// Set Trace in the header
-	r.headers.Set("x-runn-trace", string(tj))
+	if s.grpcRunner != nil && s.grpcRunner.traceHeaderName != "" {
+		r.headers.Set(s.grpcRunner.traceHeaderName, string(tj))
+	} else {
+		// by Default
+		r.headers.Set(strings.ToLower(traceHeaderName), string(tj))
```

Comment:
```suggestion
		r.headers.Set(traceHeaderName, string(tj))
```

Lowercased internally.

https://github.com/grpc/grpc-go/blob/dbbcf59957fec0bd58063224cbf105b3b3698d4e/metadata/metadata.go#L116